File: BLL/ModelsView/RegisteredDate_VM.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BLL.ModelsView
{
    public class RegisteredDate_VM
    {
        public List<ActivityTypes_VM> activityTypes_VM { get; set; }
        public List<Countries_VM> countries_VM { get; set; }
        public List<AddressValidations_VM> addressValidation { get; set; }

    }
}

File: Electronic_Invoice/ClientApp/src/app/Models/BasicData/RegisteredDate_VM.ts
import { countries_VM } from "./countries_VM";
import { activityTypes_VM } from "./activityTypes_VM";
import { AddressValidations_VM } from "./addressValidation_VM";

export class RegisteredDate_VM {
  constructor
    (
    public countries_VM?: countries_VM[],
    public activityTypes_VM?: activityTypes_VM[],
    public AddressValidations_VM?: AddressValidations_VM[],
    ) { }
}
